src/libraries/System.Net.Http/src/System/Net/Http/DelegatingHandler.cs

```diff
@@ -7,16 +7,18 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Diagnostics.CodeAnalysis;
 
 namespace System.Net.Http
 {
     public abstract class DelegatingHandler : HttpMessageHandler
     {
-        private HttpMessageHandler _innerHandler;
+        private HttpMessageHandler? _innerHandler = null!;
```

Comment:
I presume you need to either mark as nullable or initialize here?

Reply:
yep, forgot to remove initialization after making it nullable, thanks!